target/linux/ipq40xx/base-files/etc/board.d/02_network

```diff
@@ -104,6 +104,14 @@ ipq40xx_setup_interfaces()
 			"0u@eth0" "3:lan:2" "4:lan:1"
 		ucidef_set_interface_wan "eth1"
 		;;
+	netgear,rbr50|\
+	netgear,rbs50|\
+	netgear,srr60|\
+	netgear,srs60)
+		ucidef_set_interfaces_lan_wan "eth0" "eth1"
```

Comment:
Just using ucidef_set_interface_wan should be enough, the lan should be picked up from the switch command.

Reply:
I tested with your suggested changes and it's working correclty. Thanks